components/browser/engine-gecko-nightly/build.gradle

```diff
@@ -46,11 +55,18 @@ dependencies {
     testImplementation project(':support-test')
     testImplementation project(':tooling-fetch-tests')
 
+    // We only compile against Glean. It's up to the app to add those dependencies
```

Comment:
These are required at compile time 'cuz the generated code uses the Glean SDK, right?

I don't feel strongly about this but consider making this a regular `implementation` dependency; folks who don't actually consume the generated Glean definitions should get that stuff stripped by r8/PG.

Reply:
This was a requirement by the a-c team (cc @pocmo) to not have an hard dependency on Glean at runtime, unless the product consuming 'engine-gecko' is also using Glean (which is not always the case). We're using this pattern in `lib-crash` as well. We might consider revisiting this in the future :)